### PR TITLE
[OTE_SDK_TESTS] Add check of nncf model graph

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
+ Add check of NNCF model graph
+ Removed debug timeit from ote_sdk/ote_sdk/utils/segmentation_utils.py 

TBD update submodules for:
https://github.com/openvinotoolkit/deep-object-reid/pull/167
https://github.com/openvinotoolkit/mmdetection/pull/308
https://github.com/openvinotoolkit/mmsegmentation/pull/61

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
13:55:56  isort (ote_sdk)..........................................................Failed
13:55:56  - hook id: isort
13:55:56  - files were modified by this hook
13:55:56  
13:55:56  Fixing /home/jenkins/workspace/ote/precommit/ote_sdk_tests_runner/src/ote_sdk/ote_sdk/test_suite/training_tests_actions.py
13:55:56  
13:55:56  isort (ote_cli|external).................................................Passed
13:55:58  black (ote_sdk|ote_cli)..................................................Failed
13:55:58  - hook id: black
13:55:58  - files were modified by this hook
13:55:58  
13:55:58  reformatted ote_sdk/ote_sdk/test_suite/fixtures.py
13:55:58  reformatted ote_sdk/ote_sdk/test_suite/training_tests_actions.py
13:55:58  All done! ✨ 🍰 ✨
13:55:58  2 files reformatted, 227 files left unchanged.
13:55:58  
13:55:58  black (rest).............................................................Passed
13:56:01  flake8...................................................................Failed
13:56:01  - hook id: flake8
13:56:01  - exit code: 1
13:56:01  
13:56:01  ote_sdk/ote_sdk/test_suite/training_tests_actions.py:464:24: F541 f-string is missing placeholders
13:56:01  
13:56:01  prettier.................................................................Passed
13:56:06  mypy (ote_sdk)...........................................................Passed
13:56:07  mypy (ote_cli)...........................................................Passed
13:56:08  mypy (external)..........................................................Passed
13:56:34  pylint...................................................................Passed
13:56:35  [Pipeline] }
13:56:35  ERROR: script returned exit code 1

**Comment by AlexanderDokuchaev** [comment]
run ote_sdk tests

**Comment by AlexanderDokuchaev** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/test_suite/training_tests_actions.py [hunk 1]
```diff
@@ -490,6 +542,10 @@ def _run_ote_nncf(
         assert (
             self.nncf_model.model_format == ModelFormat.BASE_FRAMEWORK
         ), "Wrong model format"
+        assert check_nncf_model_graph(
```
Comment (alexsu52) [inline ote_sdk/ote_sdk/test_suite/training_tests_actions.py:545]: @AlexanderDokuchaev Could you create a separate test to check the compression graph?

**Pull request merged**